tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_SUBCONUS_Ind_3km_ics_NAM_lbcs_NAM_suite_GFS_v16.yaml

```diff
@@ -8,16 +8,16 @@ user:
 workflow:
```

Comment:
Should the description be updated to note that ICs and LBCs are derived from NAM rather than FV3GFS?